apps/backend/.env.example

```diff
@@ -12,3 +12,10 @@ DB_CONNECTION_STRING=[db_client]://[db_username]:[db_user_password]@localhost:[d
 DB_DIALECT=pg
 DB_POOL_MIN=2
 DB_POOL_MAX=10
+
+#
+# JWT
+#
+SECRET=aea53992-b726-4a2e-b3c0-79a728944976
```

Comment:
Was the value for SECRET added intentionally? It seems that we should keep it private and replace the actual value with with <UUID>. What do you think?

Reply:
If you mean to replace it with . In env.example I will do that

Reply:
yep, replace it with <UUID> in .env.example